apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/avtale/Trygdeavtale.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.etterlatte.trygdetid.avtale
+
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import java.util.*
+import java.util.UUID.randomUUID
+
+data class Trygdeavtale(
+    val id: UUID = randomUUID(),
+    val behandlingId: UUID,
+    val avtaleKode: String,
+    val avtaleDatoKode: String?,
+    val avtaleKriteriaKode: String?,
+    val kilde: Grunnlagsopplysning.Saksbehandler
+)
+
+data class TrygdeavtaleRequest(
```

Comment:
Burde denne ut i DTO-land i `etterlatte-trygdetid-model` eller kommer den ikke til å bli delt til noen andre

Reply:
Selve request er kun til bruk for lagring fra beregningssiden.

Men - jeg tror du har et godt poeng på selve Trygdeavtale. Vi er usikker på hvor det skal brukes videre - men - det er nok sansynlig at det skal i -model - flyttes.